README.md

```diff
@@ -57,6 +57,37 @@ Once you have installed the credential helper, see the
 [Configuration section](#Configuration) for instructions on how to configure
 Docker to work with the helper.
 
+### Debian Buster (testing) and Debian Sid (unstable)
+You can install the Amazon ECR Credential Helper from the Debian Buster or
+Debian Sid archives.
+
+[![Debian Testing package](https://repology.org/badge/version-for-repo/debian_testing/amazon-ecr-credential-helper.svg)](https://repology.org/metapackage/amazon-ecr-credential-helper/versions)
+[![Debian Unstable package](https://repology.org/badge/version-for-repo/debian_unstable/amazon-ecr-credential-helper.svg)](https://repology.org/metapackage/amazon-ecr-credential-helper/versions)
+
+```bash
+$ sudo apt update
+$ sudo apt install amazon-ecr-credential-helper
+```
+
+Once you have installed the credential helper, see the
+[Configuration section](#Configuration) for instructions on how to configure
+Docker to work with the helper.
+
+### Ubuntu 19.04 Disco Dingo
```

Comment:
Similar to Debian, I'd try to pick language that'll continue to be meaningful as future releases go. We can expect the package to be installable on 19.10, but this language will be broken. Maybe "19.04 and newer" or "19.04+" or something like that.

Reply:
As above

Reply:
Done